.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "3rdParty/cryptography"]
```

Comment:
Can we rename `3rdParty` to `external`? I think that suits the crypto lib a bit more.